tests/test_aliases.py

```diff
@@ -0,0 +1,291 @@
+import gino
+import pytest
+import urllib.parse
+
+
+@pytest.mark.parametrize(
+    "guid,aliases,is_post",
+    [
+        ("test_get_aliases", ["alias_a"], True),
+        ("test_get_aliases", ["alias_a"], False),
```

Comment:
you could use a 2nd parameter to apply `is_post=true/false` to each `guid,aliases` value

Reply:
I'm not sure I fully understand what you're suggesting. The `is_post` is already an additional parameter that's applied true/false to each guid,alias

Reply:
i haven't tested it so maybe i'm misremembering, but i think instead of:
```
@pytest.mark.parametrize(
    "guid,aliases,is_post",
    [
        ("1", ["alias_a"], True),
        ("1", ["alias_a"], False),
        ("2", ["alias_b"], True),
        ("2", ["alias_b"], False),
    ]
)
```
you could do:
```
@pytest.mark.parametrize(
    "guid,aliases",
    [
        ("1", ["alias_a"]),
        ("2", ["alias_b"]),
    ]
)
@pytest.mark.parametrize("is_post", [True, False])
```
since you always want to test both value of `is_post`

Reply:
ohhh interesting, I'll give that a shot, that is definitely cleaner

Reply:
yeah good call out, this is much nicer. pushing an update now